### PR TITLE
update DEV to current RC state

### PR DESCRIPTION
**Pull request by croqueGrec09**

### CODE REVIEW

File: documentation/CHANGELOG.md [hunk 1]
```diff
@@ -237,143 +255,143 @@
...
-5145    19.06.2023  rc-3.1  3.1.3       Andreas Bug         Export der Identifier bei Titelexport griff nicht
+5145    19.06.2023  rc-3.1  3.1.2       Andreas Bug         Export der Identifier bei Titelexport griff nicht
 
-5144    19.06.2023  rc-3.1  3.1.3       Andreas Bug         Überprüfung und Korrektur von Synchronisationsabläufen und Bereinigungsmethoden
+5144    19.06.2023  rc-3.1  3.1.2       Andreas Bug         Überprüfung und Korrektur von Synchronisationsabläufen und Bereinigungsmethoden
 
-5142    19.06.2023  rc-3.1  3.1.3       Moe     Bug         Übertrag Teilnehmer
+5142    19.06.2023  rc-3.1  3.1.2       Moe     Bug         Übertrag Teilnehmer
 
 4545    16.06.2023  rc-3.2  3.2         Ingrid  Feature     Bestands-View: Filterergebnis "Stichtag" verbessern
```
Comment (klodav) [inline documentation/CHANGELOG.md:282]: rc-3.2 / 3.2 ?

File: documentation/CHANGELOG.md [hunk 2]
```diff
@@ -115,13 +133,13 @@
 
 --      01.08.2023  rc-3.2  3.2.2       Andreas Bug         Exportreiter "Übertrag" hat keine Defaultbelegung mehr
```
Comment (klodav) [inline documentation/CHANGELOG.md:134]: rc-3.2 / 3.2.2 ?

File: documentation/CHANGELOG.md [hunk 3]
```diff
@@ -31,81 +45,85 @@
...
+5279    13.09.2023  rc-3.2  3.2.4       Andreas Bug         Dauerhafte Titel wurden nicht verzeichnet
+
+--      12.09.2023  rc-3.2  3.2.4       Andreas Bug         fehlende Blockade bei hart kodierten Namensräumen eingefügt
+
+5217    08.09.2023  rc-3.2  3.2.4       Moe     Bug         Anlegen von Dokumenten beim Übertrag und Lizenzübertragung fehlerhaft
 
-5271    08.09.2023  rc-3.2  3.2.5       Andreas Bug         diverse Denk- und Programmierfehler bei der Titelverwaltung behoben
+5271    08.09.2023  rc-3.2  3.2.4       Andreas Bug         diverse Denk- und Programmierfehler bei der Titelverwaltung behoben
 
 5241    08.09.2023  rc-3.2  3.2-5       Moe     Feature     Meine Lizenzübertragungen: neue Spalte "Reminder"
```
Comment (klodav) [inline documentation/CHANGELOG.md:56]: rc-3.2 / 3.2-5 ?

File: documentation/CHANGELOG.md [hunk 4]
```diff
@@ -31,81 +45,85 @@
 
 --      12.09.2023  rc-3.2  3.2.5       Andreas Bug         fehlende Blockade bei hart kodierten Namensräumen eingefügt
```
Comment (klodav) [inline documentation/CHANGELOG.md:46]: rc-3.2 / 3.2.5 ?

**Review by klodav — COMMENTED** [review]
?

File: documentation/CHANGELOG.md (hunk 2 above)
Comment (croqueGrec09) [reply documentation/CHANGELOG.md:134]: fixed

File: documentation/CHANGELOG.md (hunk 3 above)
Comment (croqueGrec09) [reply documentation/CHANGELOG.md:56]: typo fixed

File: documentation/CHANGELOG.md (hunk 1 above)
Comment (croqueGrec09) [reply documentation/CHANGELOG.md:282]: fixed

**Review by klodav — APPROVED** [review]

**Pull request merged**